.gitignore

```diff
@@ -1,2 +1,3 @@
 dist
 node_modules
+.vscode
```

Comment:
@jpramos123 not a big deal but just FYI there is a global .gitignore file that can be setup to ignore your IDE specific files instead of defining them for each project, https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer